gzip.opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "jonathan.ludlam@eu.citrix.com"
```

Comment:
are you sure we want Jon to maintain all these? or should we use the `lists.xen.org` address instead?

Reply:
I don't think we will publish these anyway, but I can change that to be consistent with the other libraries